helpers/measures.ts

```diff
@@ -56,6 +56,23 @@ export function isDiffMetric(metricKey: string): boolean {
   return metricKey.indexOf('new_') === 0;
 }
 
+/*
+ * Conditional decimal count for QualityGate-impacting measures
+ * (e.g. Coverage %)
+ */
+export function getDecimalsNumber(threshold: number | undefined, value: number) {
+  if (!threshold) {
+    return undefined;
+  }
+  const delta = Math.abs(threshold - value);
+  if (delta < 0.1 && delta > 0) {
+    const match = delta.toFixed(20).match('[^0.]');
+    return match && match.index ? match.index - 1 : undefined;
+  } else {
+    return undefined;
```

Comment:
Why not 0 in this case ?

Reply:
to fallback to default

Reply:
So my issue is `getDecimalsNumber` which returns something else than a number.

Reply:
That's a faire point, I'll rename it along with the better comment